src/entity_embedding_utils.jl

```diff
@@ -0,0 +1,148 @@
+"""
+A file containing functions or constants used in the `fit` and `update` methods in `mlj_model_interface.jl` for setups supporting entity embeddings
+"""
+
+EMBEDDING_ENABLED_MODELS = [
+    NeuralNetworkClassifier,
+    NeuralNetworkBinaryClassifier,
+    NeuralNetworkRegressor,
+    MultitargetNeuralNetworkRegressor,
+]
+
+EMBEDDING_ENABLED_MODELS_UNION = Union{EMBEDDING_ENABLED_MODELS...}
+
+
+# A function to check if a model is among those in EMBEDDING_ENABLED_MODELS
+function is_embedding_enabled_type(model_type)
+    return any(model_type <: T for T in EMBEDDING_ENABLED_MODELS)
+end
```

Comment:
This is kind of brittle and not the Julian way of doing this. You could add a new abstract type for models that implement entity-embedding, but I suggest you instead  add "is_empty_enabled_type" as a new trait for the local API:

In `mlj_model_interface.jl` do this:

```julia
is_empty_enabled_type(model) = false
```

Then, for each model that supports entity embedding, just overload, as in this example:

```julia
is_embedding_enabled_type(::NeuralNetworkRegressor) = true
```

Although, if you get rid of case-distinctions as I suggest in another comment, you may not even need this trait at all. 


```suggestion
```

Reply:
Done in e5d8141fd2faac36619fdfb72d452960283d4d95

Reply:
I did this and then got rid of the case distinction. See f60f789acb0b02452565a4654619f133be6fd60b